src/Core/LanguageService.fs

```diff
@@ -675,26 +675,39 @@ Consider:
                     |> Option.map fst
                     |> Option.defaultWith (fun () -> Seq.last availableTFMs)
 
-            let fsacPathForTfm (tfm: string) =
-                if String.IsNullOrEmpty fsacNetcorePath then
-                    let binPath = node.path.join (VSCodeExtension.ionidePluginPath (), "bin")
+            let probePathForTFMs (basePath: string) (tfm: string) =
+                let availableTFMs =
+                    node.fs.readdirSync (!!basePath)
+                    |> Seq.filter (fun p -> p.StartsWith "net") // there are loose files in the basePath, ignore those
+                    |> Seq.map node.path.basename
 
-                    let availableTFMs =
-                        node.fs.readdirSync (!!binPath)
-                        |> Seq.filter (fun p -> p.StartsWith "net") // there are loose files in the binpath, ignore those
-                        |> Seq.map node.path.basename
+                printfn $"Available FSAC TFMs: %A{availableTFMs}"
 
-                    printfn $"Available FSAC TFMs: %A{availableTFMs}"
+                if availableTFMs |> Seq.contains tfm then
+                    printfn "TFM match found"
+                    node.path.join (basePath, tfm, "fsautocomplete.dll")
+                else
+                    // find best-matching
+                    let tfm = findBestTFM availableTFMs tfm
+                    node.path.join (basePath, tfm, "fsautocomplete.dll")
```

Comment:
pulled this probing out of the `fsacPathForTfm` member so it could be retargeted against whatever base path we wanted it to be.